CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# v2.1.0
+
+Reaction v2.1.0 adds minor features and performance enhancements, fixes bugs and contains no breaking changes on top of v2.0.0.
+
+This release is being coordinated with `reaction-platform` and is designed to work with the same versions of `example-storefront` and `reaction-hydra`.
```

Comment:
is `...designed to work with v2.1.0...` rather than `...designed to work with the same versions of...`